xdsl/frontend/pypdl/builder.py

```diff
@@ -0,0 +1,19 @@
+from dataclasses import dataclass, field
+
+from xdsl.frontend.pyast.utils.builder import PyASTBuilder
+from xdsl.frontend.pypdl.transforms.func_to_pdl_rewrite import FuncToPdlRewrite
+from xdsl.passes import ModulePass
+from xdsl.transforms.desymref import FrontendDesymrefyPass
+
+
+@dataclass
+class PyPDLRewriteBuilder(PyASTBuilder):
```

Comment:
Would passing post_transforms and post_verify as arguments work? I don't see advantages to subclassing in this case.